indexed_gzip/indexed_gzip.pyx

```diff
@@ -289,6 +289,10 @@ cdef class _IndexedGzipFile:
 
         return proxy()
 
+    def points(self):
```

Comment:
How about naming the method something a little more descriptive, e.g. `def seek_points`?